### ISSUE TITLE
Better VAD: py-webrtcvad

### ISSUE THREAD

**Issue by readbeyond** [readbeyond/aeneas#101]
This VAD, conveniently available on PyPI, is a Python wrapper around the VAD from WebRTC by Google:

https://github.com/wiseman/py-webrtcvad

From preliminary tests it works well and it is fast. Its API might need some adaptation and/or we can create our own C extension.

**Comment by readbeyond** [comment]
Deferring this, since the library only supports 10, 20, or 30 ms frames and this makes things more complex than initially imagined.

**Comment by readbeyond** [comment]
It actually have some issues related to packaging, dropping for the time being.

**Comment by cbenhagen** [comment]
What are the issues? Can I help?

**Comment by readbeyond** [comment]
Hi,

thanks for your interest.

I did some experiments with it, and --- when integrated into aeneas --- 
did not yield significant improvements to the resulting alignment than 
the simple, energy-based VAD algorithm included in aeneas. But, to be 
honest, I did not spend too much time on it.

At most, for aeneas 2, I can consider shipping the current simple VAD, 
but also offering the possibility of using other VADs, with glue code 
written by the end-user (or contributed and included in the aeneas PyPI 
package), shifting the responsibility of installing the desired VAD 
library/package to the end user --- in the same way it is now done for 
TTS engines other than eSpeak.

AP
